db/seeders/data/pending-applications.js

```diff
@@ -0,0 +1,129 @@
+        bankSortCode: faker.finance.account(6),
+        bankStatement: {
+            size: 123,
+            type: "application/pdf",
+            filename: "example.pdf"
+        },
+        buildingSocietyNumber: faker.random.alphaNumeric(8),
+        beneficiariesGroupsCheck: 'no',
+        beneficiariesGroupsReligionOther: "",
+        charityNumber: faker.finance.account(7),
+        mainContactName: {
+            lastName: faker.name.lastName(),
+            firstName: faker.name.firstName()
+        },
+        mainContactEmail: faker.internet.email(),
+        mainContactDateOfBirth: toDateParts(moment().subtract(19, 'years')),
+        mainContactPhone: "0345 4 10 20 30",
+        mainContactAddress: mockAddress(),
+        mainContactAddressHistory: {
+            currentAddressMeetsMinimum: "yes"
+        },
+        mainContactCommunicationNeeds: "",
+        organisationLegalName: faker.company.companyName(),
+        organisationTradingName: faker.company.companyName(),
+        organisationType: 'unincorporated-registered-charity',
+        organisationStartDate: {
+            year: 2017,
+            month: 11,
+            isBeforeMin: true
+        },
+        organisationAddress: mockAddress(),
+        projectName: faker.lorem.words(5),
+        projectBudget: [
+            {
+                cost: faker.random.number({ min: 100, max: 300 }),
+                item: faker.lorem.words(5)
+            }
+        ],
+        projectCountry: faker.random.arrayElement(projectCountries),
+        projectLocation: "east-lothian",
+        projectLocationDescription: "retr",
+        projectPostcode: mockAddress().postcode,
+        projectDateRange: {
+            endDate: toDateParts(moment().add(30, 'weeks')),
+            startDate: toDateParts(moment().add(18, 'weeks'))
+        },
+        projectTotalCosts: faker.random.number({ min: 301, max: 400 }),
+        seniorContactName: {
+            lastName: faker.name.lastName(),
+            firstName: faker.name.firstName()
+        },
+        seniorContactEmail: faker.internet.email(),
+        seniorContactDateOfBirth: toDateParts(moment().subtract(22, 'years')),
+        seniorContactPhone: "0345 4 10 20 30",
+        seniorContactAddress: mockAddress(),
+        seniorContactAddressHistory: {
+            currentAddressMeetsMinimum: "yes"
+        },
+        seniorContactCommunicationNeeds: "",
+        seniorContactRole: "trustee",
+        termsPersonPosition: faker.name.jobTitle(),
+        termsPersonName: `${faker.name.firstName()} ${faker.name.lastName()}`,
+        totalIncomeYear: faker.random.number({ min: 1000, max: 1000000 }),
+        yourIdeaCommunity: faker.lorem.words(55),
+        yourIdeaPriorities: faker.lorem.words(55),
+        yourIdeaProject: faker.lorem.words(55)
+    };
+};
+
+const prepareSeeds = async () => {
+    const user = await Users.createUser({
+        username: faker.internet.email(),
```

Comment:
Tweak here, this generates potentially real email addresses (ran this and got a `@gmail.com` address) for other user seeds we've used `${uuid}@example.com` for the email address.